src/main/webapp/js/feedbackResponseComments.es6

```diff
@@ -347,25 +347,28 @@ function registerResponseCommentsEvent() {
 
     const clickHandlerMap = new Map();
     clickHandlerMap.set('.show-frc-add-form',
-            [showResponseCommentAddForm, ['recipientindex', 'giverindex', 'qnindex', 'sectionindex']]);
+        [showResponseCommentAddForm, ['recipientindex', 'giverindex', 'qnindex', 'sectionindex']]);
```

Comment:
I realised that my previous comment may have been interpreted differently; by "8 spaces" I meant "8 spaces deeper indentation than the previous line", and not "8 spaces, absolutely speaking", so in absolute terms this should use 4 (from the previous line) + 8 (due to continuation of previous line) = 12 spaces from the start of the line.

This applies to the other comments as well.